chassis/util/src/main/java/com/griddynamics/jagger/util/StandardMetricsNamesUtil.java

```diff
@@ -21,6 +21,11 @@
 package com.griddynamics.jagger.util;
 
 
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static org.apache.commons.lang3.StringUtils.removePattern;
```

Comment:
will apache commons work in web client (gwt)? in the past there were issues with third party libraries and gwt